src/opm/input/eclipse/Schedule/Well/WVFPEXP.cpp

```diff
@@ -64,9 +68,18 @@ namespace Opm {
     }
 
     bool WVFPEXP::prevent() const {
-        return m_prevent != Prevent::No;;
+        return m_prevent != Prevent::No;
+    }
+
+    bool WVFPEXP::report_first() const
+    {
+        return this->m_prevent == Prevent::First;
```

Comment:
> The reads really strange: "we report first if told to prevent first". Same below.

Don't disagree.  I'm trying to capture `WVFPEXP(4) == "YES1"` here.  This item is known as [PREVENT_THP](https://github.com/OPM/opm-common/blob/2aff28dd575327af87b464ae174969ed7948f322/src/opm/input/eclipse/share/keywords/000_Eclipse100/W/WVFPEXP#L21-L25) in our datamodel.  I suppose we could rename the enumerator to `ReportFirst` or something like that.